lib/util.js

```diff
@@ -138,11 +139,16 @@ function cleanUpAssertion(element) {
 }
 
 function getAssertionComponents(element) {
-  let matched = element.match(/(.*)\/\/\s(\w+-?\w+?)(:\s(.+))?;/);
+  const regEx =/(.*)\/\/\s*(?:([a-z\-]+:|throws:?))([^;]+)?(?:.*)/i;
+  let matched = element.match(regEx);
   if(!matched) {
     return null;
   }
-  let [, code, assertion, , expected] = matched;
+  let [, code, assertion, expected] = matched;
+  let index = assertion.indexOf(':');
```

Comment:
Since we have no named grouping. We are relying on array destructuring. 
`const regEx =/(.*)\/\/\s*(?:([a-z\-]+:|throws:?))([^;]+)?(?:.*)/i;` this will give right match for following `let [, code, assertion, expected] = matched;` destructuring. 
`const regEx =/(.*)\/\/\s*(?:([a-z\-]+:)|(throws:?))([^;]+)?(?:.*)/i;` this will give right match some time have 4th element if it matches with ([a-z\-]+:) else will have 5th element if matched with (throws:?). which makes destrcuturing tough. So it is easy to slice and take out ` :`